ng2-components/ng2-activiti-tasklist/src/services/activiti-tasklist.service.ts

```diff
@@ -50,6 +50,21 @@ export class ActivitiTaskListService {
     }
 
     /**
+     * Retrieve Deployed App details by appId
+     * @returns {Observable<any>}
+     */
+    getApplicationDetailsById(appId?: string): Observable<any> {
```

Comment:
The parameter should be required not optional, please remove the character '?'
can you change the typo from string to number ?